src/utils/check-icons.tsx

```diff
@@ -0,0 +1,11 @@
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline'
+import DriveFileRenameOutlineOutlinedIcon from '@mui/icons-material/DriveFileRenameOutlineOutlined'
+import FormatListBulletedOutlinedIcon from '@mui/icons-material/FormatListBulletedOutlined'
+
+export const CheckIcons = (type: string) => {
+  return (
+    (type === 'oneAnswer' && <CheckCircleOutlineIcon />) ||
+    (type === 'openAnswer' && <DriveFileRenameOutlineOutlinedIcon />) ||
+    (type === 'multipleChoice' && <FormatListBulletedOutlinedIcon />)
```

Comment:
maybe we need to put this string into some sort of `constants` file, or is that redundant?

Reply:
yeah , that`s good idea